WordPress/Credentials/Secrets-example.swift

```diff
@@ -20,7 +20,6 @@ class ApiCredentials: NSObject {
     // Other Services
     static let tenorApiKey = ""
     static let sentryDSN = ""
-    static let appCenterAppId = ""
```

Comment:
Once this PR lands, we might want to remove that constant from the actual `Secrets-*.swift` files in `~/.mobile-secrets` ([see `.configure` setup](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/.configure#L16-L35)), as from a quick grep in the source code, this constant doesn't seem to be used anywhere in the codebase anymore.